plugins/modules/host.py

```diff
@@ -746,6 +711,7 @@ def perform_module_operation(self):
                         "provide a valid value."
             LOG.error(error_msg)
             self.module.fail_json(msg=error_msg)
```

Comment:
We can shuffle the if conditions and put this common code at end so that the code is not duplicated

Reply:
Not applicable as the code may not go in either scenario, hence no failure.